robots/go2_description/urdf/go2.urdf

```diff
@@ -36,88 +32,11 @@ Stephen Brawner (brawner@gmail.com)
       </material>
     </visual>
     <collision>
-      <origin rpy="0 0 0" xyz="0 0 0" />
-      <geometry>
-        <box size="0.3762 0.0935 0.114" />
-      </geometry>
-    </collision>
-  </link>
-  <link
-    name="Head_upper">
```

Comment:
I hope this link was not used by anybody for anything else that collisions.
Technically, removing this is a breaking change, but I think we can take the risk.

Reply:
Yes, you are completely right. I thought that since the Go2 model was added recently by @edantec it was still early enough to do this kind of changes...